repose-aggregator/core/repose-core/src/test/java/org/openrepose/nodeservice/request/RequestHeaderServiceContextTest.java

```diff
@@ -55,8 +52,6 @@
 import static org.mockito.Matchers.eq;
 import static org.mockito.Mockito.*;
 
-@RunWith(PowerMockRunner.class)
-@PowerMockIgnore("javax.management.*")
```

Comment:
Ok. This was originally removed because it popped up on a case insensitive search for `MockRunner`. Then after it was removed, it still worked.
Any idea why it was here in the first place?